.circleci/config.yml

```diff
@@ -4,9 +4,15 @@ jobs:
   build:
     working_directory: /go/src/github.com/resin-io/sshproxy
     docker:
-      - image: golang:1.8.3
+      - image: golang:1-alpine
     steps:
       - checkout
+      - run:
+          name: Install dependecies
+          command: apk add --no-cache build-base git
+      - run:
```

Comment:
This seems superfluous.

Reply:
@wrboyce Alpine go didn't have these dependencies installed.

Reply:
Or which part you are referring to? Maybe the version print? That's the thing to not pin to specific version and still see what version it was built with.

Reply:
Sorry yeah, I meant the version printing :)